docs/reference/commandline/service_create.md

```diff
@@ -113,7 +113,7 @@ $ docker service create \
 When this service is [updated](service_update.md), a rolling update will update
 tasks in batches of `2`, with `10s` between batches.
 
-### Setting environment variables (-e --env)
+### Set environment variables (-e --env)
```

Comment:
Since you touched this line, I think it's fair game to ask if there's a clearer way to mention that `-e` and `--env` are synonyms.

`-e or --env`?
`-e / --env`?